### PR TITLE
Chore: Convert diagrams to mermaid.js

### PR DESCRIPTION
**Pull request by lena-larionova**
### Description

Converted a few Kong Gateway and Konnect diagrams into mermaid.js format.

### Testing instructions

Preview links:
https://deploy-preview-7000--kongdocs.netlify.app/gateway/latest/get-started/services-and-routes/
https://deploy-preview-7000--kongdocs.netlify.app/gateway/latest/get-started/load-balancing/
https://deploy-preview-7000--kongdocs.netlify.app/gateway/latest/production/deployment-topologies/hybrid-mode/
https://deploy-preview-7000--kongdocs.netlify.app/konnect/gateway-manager/control-plane-groups/

### Checklist 

- [x] Review label added 
- [x] [Conditional version tags](https://docs.konghq.com/contributing/conditional-rendering/#conditionally-render-content-by-version) added, if applicable.

For example, if this change is for an upcoming 3.6 release, enclose your content in `{% if_version gte:3.6.x %} <content> {% endif_version %}` tags (or `if_plugin_version` tags for plugins). 

Use any of the following keys:
* `gte:<version>` - greater than or equal to a specific version
* `lte:<version>` - less than or equal to a specific version
* `eq:<version>` - exactly equal to a specific version

You can do the same for older versions.

### CODE REVIEW

File: app/_src/gateway/get-started/services-and-routes.md [hunk 1]
```diff
@@ -14,8 +14,35 @@ through the system.
...
+
+{% mermaid %}
+flowchart LR
+  A(API client)
+  B("`Route 
+  (/mock)`")
+  C("`Service
+  (example_service)`")
+  D(Backend/
+  upstream API)
```
Comment (Guaris) [inline app/_src/gateway/get-started/services-and-routes.md:28]:
```suggestion
  D(Upstream application)
```

Comment (Guaris) [reply app/_src/gateway/get-started/services-and-routes.md:28]: the doc uses 'upstream application', it feels like we should use that in the diagram too.

File: app/_src/gateway/get-started/load-balancing.md [hunk 2]
```diff
@@ -24,7 +24,31 @@ instead of a specific host. For the purposes of our example, the upstream will p
...
+  (example_service)`")
+  C(Load balancer)
+  D(httpbin.org)
+  E(httpbun.com)
+  
+  subgraph id1 ["`**KONG GATEWAY**`"]
+    A --> B --> C
+  end
+
+  subgraph id2 ["`Targets (example_upstream)`"]
```
Comment (Guaris) [inline app/_src/gateway/get-started/load-balancing.md:43]: Is it possible to have targets appear on top of example_upstream?

File: app/_src/gateway/production/deployment-topologies/hybrid-mode/index.md [hunk 3]
```diff
@@ -15,7 +15,38 @@ of the CP nodes, and only the CP nodes are directly connected to a database.
 Instead of accessing the database contents directly, the DP nodes maintain a 
 connection with CP nodes to receive the latest configuration.
 
-![Hybrid mode topology](/assets/images/products/gateway/deployment-hybrid-2.png)
+<!--vale off -->
+{% mermaid %}
+flowchart LR
+
+A[(Database)]
```
Comment (Guaris) [inline app/_src/gateway/production/deployment-topologies/hybrid-mode/index.md:22]: I think the use of 'kong gateway instance' isn't as helpful as Data plane 1, 2, 3 and control plane.

**Review by Guaris — COMMENTED** [review]
Tiny nits. I really think this one is like a work of art
https://deploy-preview-7000--kongdocs.netlify.app/konnect/gateway-manager/control-plane-groups/

File: app/_src/gateway/get-started/load-balancing.md (hunk 2 above)
Comment (lena-larionova) [reply app/_src/gateway/get-started/load-balancing.md:43]: Unfortunately no.  This is what happens when you try to add a line 
Screenshot 2024-02-28 at 1 00 56 PM
break to a title:

**Review by Guaris — APPROVED** [review]

**Pull request merged**